core/src/loader.rs

```diff
@@ -42,6 +42,8 @@ use url::{form_urlencoded, ParseError, Url};
 
 pub type Handle = Index;
 
+const LOADER_INSERTED_AVM1_DEPTH: i32 = -0xF000;
```

Comment:
A short, one-line comment on what this is would be nice. It's almost self-explanatory, but still.

Reply:
Would you mind if I added this myself before merging, @Lord-McSweeney?
`/// This is the depth where AVM2 puts the AVM1 movies it loads.`
Is this even correct?